ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -261,6 +339,7 @@ export default class ExperimentalTab extends PureComponent {
   render() {
     return (
       <div className="settings-page__body">
+        {this.renderSecurityAlertsToggle()}
```

Comment:
I think the code is looking good! I think there are a couple of advancements we need to add 

1. In one meeting (I believe not recorded), @jpuri mentioned adding the code behind code fences `build.yml`. The last time I heard, the plan is to introduce blockaid as a flask feature. I think we'll need to add this to this PR.

2. If we are rendering the "Security alerts" section, we will want to hide the "Enable security alerts" section. Current: 
<img width="664" alt="Screen Shot 2023-07-13 at 12 02 01 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/120b4532-38c3-4bea-939b-fe6ea92475b9">

Reply:
@digiwand Number 2 will be done in another PR...there's another issue for it. [here](https://github.com/MetaMask/MetaMask-planning/issues/926)

I'll add the code fencing. Thanks.